ui/src/institutions/containers/SearchPageContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col } from 'antd';
+import { connect } from 'react-redux';
+
+import PaginationContainer from '../../common/containers/PaginationContainer';
+import ResultsContainer from '../../common/containers/ResultsContainer';
+import NumberOfResultsContainer from '../../common/containers/NumberOfResultsContainer';
+import LoadingOrChildren from '../../common/components/LoadingOrChildren';
+import DocumentHead from '../../common/components/DocumentHead';
+import { INSTITUTIONS_NS } from '../../reducers/search';
+import { SEARCH_PAGE_GUTTER } from '../../common/constants';
+import InstitutionItem from '../components/InstitutionItem';
+
+const META_DESCRIPTION = 'Find institutions in High Energy Physics';
```

Comment:
was this discussed?, because it's better to put sth longer with more keyword if we care about SEO

Reply:
no, I just saw it was added the same for conferences. I also don't know what to add to the meta description in the detail page :/

Reply:
Ask @StellaCh , maybe she has something better.